rktl_control/nodes/controller

```diff
@@ -136,6 +136,9 @@ class Controller(object):
 
     def odom_cb(self, odom_msg):
         """Callback for odom messages from car."""
+        if self.vr_ref is None or self.psi_ref is None:
```

Comment:
Is this common? Do we want to throw a warning, just so that we aren't oblivious to a bunch of messages being dropped.

Reply:
This is just so that the controller is disabled until we give it an input. Otherwise it'll bounce around holding zero velocity when we first turn it on and the filters are still initializing.

Reply:
Oh I see. When I first skimmed through I thought you were referencing variables in the msg, but this makes more sense.